HowDryIAM/src/howdryiam/HowDryIAM.java

```diff
@@ -0,0 +1,62 @@
+package howdryiam;
+import java.util.Scanner;
+
+public class HowDryIAM {
+  public static void main(String[] args) {
+	Scanner keyboard = new Scanner(System.in);
+	
+	LabelPrinter label1;
+	LabelPrinter label2;
```

Comment:
you could declare these down on lines 13 and 17 instead of at the top of the block